kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/DropboxAuthzHandler.java

```diff
@@ -314,4 +314,17 @@ private static String getDropboxOwner(String entityId)
 		// parts: "", "group-user", "siteId", "userId", ...
 		return parts[3];
 	}
+	
+	private static boolean isSiteMember(String siteId, String userId)
+	{
+		try
+		{
+			return siteService.getSite(siteId).getMember(userId) != null;
+		}
+		catch (IdUnusedException e)
+		{
+			log.warn("IdUnusedException in isSiteMember for " + userId);
```

Comment:
better log message and use newer log pattern and don't forget to add the exception message for better logging
```suggestion
			log.warn("User [{}] is not a member of site [{}], {}", userId, siteId, e.toString());
```

Reply:
Sorry for the delay, I have been on holidays. Thanks for the review, Earle, the change is done.

Reply:
Hi @ern , do you have any other suggestion on this code? Can it be merged? Thanks!